_source/api/logzio-public-api.yml

```diff
@@ -3711,6 +3743,27 @@ definitions:
       pagination:
         $ref: '#/definitions/Pagination'
 
+  LookupListElementBulkResponse:
+    type: object
+    properties:
+      status:
+        type: string
+        enum:
+          - SUCCESS
+          - PARTIAL_FAILED
+          - FAILED
+        description: Returns the status of the requests.
+        example: SUCCESS
+      numOfElements:
+        type: integer
+        format: int32
+        description: Returns the number of elements added to the lookup list.
```

Comment:
it's the total number of elements in the array
to get the added it would be (numOfElements - numUpdated)